python/console/console_output.py

```diff
@@ -332,6 +338,13 @@ def enteredSelected(self):
         self.shell.insertFromDropPaste(cmd)
         self.shell.entered()
 
+    def searchPyQGIS(self):
+        if self.hasSelectedText():
```

Comment:
there is actually a way to get the actual token corresponding to a cursor position from QScintilla -- specifically "GetWordFromPosition"